cartoframes/data/observatory/catalog/catalog.py

```diff
@@ -58,86 +58,54 @@ class Catalog(object):
-            from cartoframes.data.observatory import Provider
-
-            Category.get('demographics')
-            Country.get('usa')
-            Provider.get('mrli')
+    >>> Category.get('demographics')
+    >>> Country.get('usa')
+    >>> Provider.get('mrli')
 
     Examples:
         The preferred way of discover the available datasets in the Catalog is through nested filters
 
-        .. code::
-
-            from cartoframes.data.observatory import Catalog
-
-            catalog = Catalog()
-            catalog.country('usa').category('demographics').datasets
+        >>> catalog = Catalog()
+        >>> catalog.country('usa').category('demographics').datasets
 
         You can include the geography as part of the nested filter like this:
 
-        .. code::
-
-            from cartoframes.data.observatory import Catalog
-
-            catalog = Catalog()
-            catalog.country('usa').category('demographics').geography('ags_blockgroup_1c63771c').datasets
+        >>> catalog = Catalog()
+        >>> catalog.country('usa').category('demographics').geography('ags_blockgroup_1c63771c').datasets
 
         If a filter is already applied to a Catalog instance and you want to do a new hierarchical search,
         clear the previous filters with the `Catalog().clear_filters()` method:
 
-        .. code::
-
-            from cartoframes.data.observatory import Catalog
-
-            catalog = Catalog()
-            catalog.country('usa').category('demographics').geography('ags_blockgroup_1c63771c').datasets
-            catalog.clear_filters()
-            catalog.country('esp').category('demographics').datasets
+        >>> catalog = Catalog()
+        >>> catalog.country('usa').category('demographics').geography('ags_blockgroup_1c63771c').datasets
+        >>> catalog.clear_filters()
+        >>> catalog.country('esp').category('demographics').datasets
 
         Otherwise the filters accumulate and you'll get unexpected results.
 
         During the discovery process, it's useful to understand the related metadata to a given Geography or Dataset.
         A useful way of reading or filtering by metadata values consists on converting the entities to a pandas
         DataFrame:
 
-        .. code::
-
-            from cartoframes.data.observatory import Catalog
-
-            catalog = Catalog()
-            catalog.country('usa').category('demographics').geography('ags_blockgroup_1c63771c').datasets.to_dataframe()
+        >>> catalog = Catalog()
+        >>> catalog.country('usa').category('demographics').geography('ags_blockgroup_1c63771c').datasets.to_dataframe()
 
         For each dataset in the Catalog, you can explore its variables, get a summary of its stats, etc.
 
-        .. code::
-
-            from cartoframes.data.observatory import Dataset
-
-            dataset = Dataset.get('od_acs_13345497')
-            dataset.variables()
+        >>> dataset = Dataset.get('od_acs_13345497')
+        >>> dataset.variables()
```

Comment:
Isn't it `dataset.variables` since `variables` is a `@property`?

Reply:
True